torch/csrc/distributed/rpc/py_rref.cpp

```diff
@@ -152,6 +152,13 @@ PyRRef PyRRef::unpickle(const py::tuple& t) {
   return PyRRef(std::move(rref));
 }
 
+PyRRef PyRRef::local(const py::object& value) {
```

Comment:
We do we have a static method like this instead of having a constructor?

Reply:
The constructor of PyRRef is not exposed to python. This is something that works on top of that constructor.

Reply:
Can't we define a constructor for PyRRef that takes a py::object and use that in py::init()?